OpenEdXMobile/src/test/java/org/edx/mobile/util/VersionTest.java

```diff
@@ -148,4 +148,35 @@ public void testCompareTo_withMorePreciseEarlierVersion_isGreaterThan() throws P
     public void testCompareTo_withMorePreciseLaterVersion_isLessThan() throws ParseException {
         assertThat(new Version("1")).isLessThan(new Version("1.0.1"));
     }
+
+    /**
+     * Verify that the {@link Version#isNMinorVersionsDiff(Version, int)} method returns true
+     * if passed another instance with greater or equal minor versions difference than the specified
+     * value.
+     */
+    @Test
+    public void testIsNMinorVersionsDiff_withGreaterOrEqualDiff_isTrue() throws ParseException {
+        final Version version = new Version("2.2.9");
+        assertThat(version.isNMinorVersionsDiff(new Version("2.0.1"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("2.4.9"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("2.4.1"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("2.8.0"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("3.2.9"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("3.0"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("2"), 2)).isEqualTo(true);
+        assertThat(version.isNMinorVersionsDiff(new Version("4"), 2)).isEqualTo(true);
```

Comment:
* Some of these values are redundant.
* Tests are only testing difference of 2. There should be tests for negative number, 0, and another positive.
* True tests could be combined with False tests
* You should be able to use the Parameterized class for using an Array of values.